__tests__/components/viewers/__snapshots__/stop-viewer.js.snap

```diff
@@ -4746,7 +4746,7 @@ exports[`components > viewers > stop viewer should render with TriMet transit in
                         onClick={[Function]}
                       >
                         <button
-                          className="styled__Button-p56b41-2 XBKVp"
+                          className="styled__Button-p56b41-2 bjZNPA"
```

Comment:
oh my, we need to figure out how to remedy these changing snapshots ASAP. There has to be some kind of jest plugin for this.

Reply:
I have reverted the snapshot. It's a good ideas to avoid changes if the file is irrelevant.